client/js/components/statement/publicStatementModal/formGroups/FormGroupMapReference.vue

```diff
@@ -68,6 +68,7 @@
         'c-statement__formblock layout__item height-90 height-auto-lap-down u-3-of-10 u-1-of-1-palm'
       ]">
       <dp-radio
+        v-if="hasPermission('field_statement_county')"
```

Comment:
Shouldn't we move this permission check to the `div` wrapping `dp-radio` and the `select`? (And the `v-if="hasPermission('field_statement_location')"` should probably be moved to the `fieldset`, but maybe we don't want to touch it in this PR)

Reply:
Yes, you're right, it didn't make sense this way. Done in [58dc62b](https://github.com/demos-europe/demosplan-core/pull/597/commits/58dc62befff7949c9a9754836897438756b7cd7d)